hadoop-ozone/dist/src/main/smoketest/snapshot/snapshot-acls.robot

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test for reading snapshots with ACLs as different users
+Library             OperatingSystem
+Resource            snapshot-setup.robot
+Test Timeout        5 minutes
+Suite Setup         Run Keyword if  '${SECURITY_ENABLED}' == 'false'    BuiltIn.Skip
+
+*** Variables ***
+${USER1} =              testuser
+${USER2} =              testuser2
+${KEY}
+${FIRST_SNAPSHOT}
+${SECOND_SNAPSHOT}
+
+*** Keywords ***
+Add ACL
+    [arguments]         ${object}   ${user}     ${objectName}
+    ${result} =         Execute     ozone sh ${object} addacl -a user:${user}/scm@EXAMPLE.COM:a ${objectName}
+    Should not contain  ${result}   Failed
```

Comment:
The general error message doesn't have `Failed` in it, So even if it failed we can't be sure. For example here 
```
PERMISSION_DENIED User scm/scm@EXAMPLE.COM doesn't have WRITE_ACL permission to access volume Volume:s3v
```
Suggestion would be change it to `PERMISSION_DENIED`